test/kubemark/resources/hollow-node_template.yaml

```diff
@@ -51,9 +51,13 @@ spec:
             fieldRef:
               fieldPath: metadata.name
         command:
-        - /bin/sh
-        - -c
-        - /kubemark --morph=kubelet --name=$(NODE_NAME) {{hollow_kubelet_params}} --kubeconfig=/kubeconfig/kubelet.kubeconfig $(CONTENT_TYPE) --alsologtostderr 1>>/var/log/kubelet-$(NODE_NAME).log 2>&1
+        - /kubemark
+        - --morph=kubelet
+        - --name=$(NODE_NAME)
+        - {{hollow_kubelet_params}}
+        - --kubeconfig=/kubeconfig/kubelet.kubeconfig
+        - $(CONTENT_TYPE)
```

Comment:
there's another flag logtostderr default to be true. If this image uses log-file, can we also disable the logtostderr flag?